recipes/ydcpp-tcpcat/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.15)
+project(PackageTest CXX)
+
+add_executable(test_package test_package.cpp)
+
+find_package(ydcpp-tcpcat CONFIG REQUIRED)
+target_link_libraries(test_package ydcpp-tcpcat)
```

```suggestion
target_link_libraries(test_package ydcpp-tcpcat)
target_compile_features(test_package PRIVATE cxx_std_17)
```

Need to pass C++17 standard to the compiler, because in Conan 1.x compiler.cppstd is not used by default.